package pct;

message HistoryProto {
  required uint64 inst_count = 1;
  required uint64 num_threads = 2;
}

message HistoryTableProto {
  repeated HistoryProto history = 1;
}

